client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -31,10 +33,15 @@ export function normalizeRevision( revision ) {
 	}
 
 	return {
-		...revision,
+		...omit( revision, [ 'title', 'content', 'excerpt', 'date', 'date_gmt', 'modified', 'modified_gmt' ] ),
 		...flow(
 			pick( [ 'title', 'content', 'excerpt' ] ),
 			mapValues( ( val = {} ) => val.rendered )
+		)( revision ),
+		...flow(
+			pick( [ 'date_gmt', 'modified_gmt' ] ),
```

Comment:
Thinking a bit more about it, I haven't found any usage of Lodash's fp in calypso (no import, no usage inside a flow AFAICS), it might not be a good idea to start importing it (even more so if people aren't familiar with it since it can cause confusions like this one ;) and the [FP docs aren't amazing](https://github.com/lodash/lodash/wiki/FP-Guide) :( )